tests/test_all_30_scenarios.py

```
@@ -0,0 +1,387 @@
+                
+                # 404の場合はAPIエンドポイントが未実装
+                if chat_response.status_code == 404:
+                    print(f"⚠️ シナリオ{scenario_number}: チャットAPIが未実装")
+                    pytest.skip(f"シナリオ{scenario_number}のチャットAPIが未実装")
+                
+                # その他のエラー
+                if chat_response.status_code != 200:
+                    print(f"❌ シナリオ{scenario_number}: チャット失敗")
+                    print(f"   ステータス: {chat_response.status_code}")
+                    print(f"   レスポンス: {chat_response.data.decode('utf-8')[:200]}")
+                    assert False, f"シナリオ{scenario_number}のチャット失敗"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)